corehq/apps/style/templates/style/crispy/multi_inline_field.html

```diff
@@ -0,0 +1,25 @@
+{% load crispy_forms_field %}
+{% load i18n %}
+
+{% if field.is_hidden %}
+    {{ field }}
+    {% if field.errors %}
+        <div class="alert alert-danger">
+            <strong>{% trans 'Errors' %}</strong>
+            {{ field.errors }}
+        </div>
+    {% endif %}
+{% else %}
+    {% if field|is_checkbox %}
+        <div id="div_{{ field.auto_id }}" class="checkbox">
+            <label for="{{ field.id_for_label }}" class="{% if field.field.required %} requiredField{% endif %}">
```

Comment:
it's really `field.field.required`? (I see https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/bootstrap/layout/inline_field.html uses that, it's just...weird).